src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1128,6 +1110,15 @@ bool AletheProofPostprocessCallback::update(Node res,
                            {},
                            *cdp);
     }
+    case ProofRule::HO_CONG:
```

Comment:
This rule is not in the standard I think? Should it be added?

Reply:
It is not indeed. It is a cvc5-extension accepted by Carcara. We should discuss if we want to add it to the standard. For now I'd say to go ahead with having it produced by cvc5.